src/Integration.Vsix.UnitTests/CFamily/CFamilyHelperTests.cs

```diff
@@ -37,6 +39,13 @@ namespace SonarLint.VisualStudio.Integration.Vsix.CFamily.UnitTests
     public class CFamilyHelperTests
     {
         private const string FileName = @"C:\absolute\path\to\file.cpp";
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            CFamilyHelper.FileConfig.getEvaluatedPropertyValue = null;
```

Comment:
@duncanp-sonar since it's a static property, my new tests (which depend on VCConfigurationMock) would fail. So I had to reset it between tests.